helpdesk-ai/src/test/java/org/symphonyoss/symphony/bots/ai/impl/AiResponderImplTest.java

```diff
@@ -0,0 +1,33 @@
+package org.symphonyoss.symphony.bots.ai.impl;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class AiResponderImplTest {
```

Comment:
Please remove this class. It's out of scope for this PR.

Reply:
removed!